pxr/usd/usdUI/schema.usda

```diff
@@ -183,3 +188,155 @@ class Backdrop "Backdrop" (
+
+    \\note The API will use \\em default as an instance name if no namespace is specified.
+    
+    There are several best practices for using this schema.:
+
+    \\li Most accessibility runtimes support a single accessibility description. Therefore we recommend placing
+    any critical information in the default instance.
+
+    \\li A default value should be authored if using time sampled accessibility information. 
+    This helps accessibility runtimes that do not currently support time sampled information.
+
+    \\li Provide accessibility information of your scene on the default prim of the layer, and any top level prims.
+    This allows accessibility systems to provide concise scene descriptions to a user, but also allows supporting
+    accessibility systems that either do not support hierarchy information or when a user has turned off that level
+    of granularity. Accessibility information may still be provided on other prims in the hierarchy.
+
+    \\note The use of the default prim and top level prims for scene accessibility descriptions is a
+    recommended convention. Outside of that, accessibility information is not implicitly inherited through
+    a prim hierarchy. The inheritance should be left to the accessibility runtime to decide how best to
+    surface information to users.
+    """
+
+    customData = {
+        token apiSchemaType = "multipleApply"
+        token propertyNamespacePrefix = "accessibility"
+        dictionary schemaTokens = {
+            dictionary label = {
+                string value = "label"
+                string doc = """Name of the label attribute"""
+            }
+            dictionary description = {
+                string value = "description"
+                string doc = """Name of the description attribute"""
+            }
+            dictionary priority = {
+                string value = "priority"
+                string doc = """Name of the priority attribute"""
+            }
+        }
+    }
+) {
+    string label (
+        doc = """A short label to concisely describe the prim.
+        It is not recommended to time vary the label unless the concise description changes substantially."""
+    )
+
+    string description (
+        doc = """An extended description of the prim to provide more details.
+        If a label attribute is not authored in a given instance name, the description attribute should not be used
+        in it its place. A description is an optional attribute, and some accessibility systems may only use the label.
+
+        Descriptions may be time varying for runtimes that support it. For example, you may describe what a character
+        is doing at a given time.
+        """
+    )
+
+    uniform token priority = "standard" (
+        doc = """A hint to the accessibility runtime of how to prioritize this instances label and description,
+        relative to others. This attribute is optional. Accessibility runtimes may choose to apply their own
```

Comment:
Hi @dgovil.  I wanted to get some clarity on what you mean by runtimes may choose to apply their own priority systems that override this.

Are you saying that other systems can choose their own set of tokens instead of `low` / `standard` / `high`?  Or that there may be additional runtime specific attributes or configurations that may further refine the baseline priority?

If it's the former, I think I'd discourage that as it complicates validation.

Reply:
I mean that its simply a hint, and runtimes don't have to strictly stick with everything being `high` being higher priority than everything `low` if some other runtime context would otherwise prioritize the `low` state.

So they can't add their own tokens. I'll add some verbiage accordingly.